devopsloft.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
I think this should followed up with a README note to explain the meaning of the file

Reply:
+1